packages/qwik-city/middleware/request-handler/request-event.ts

```diff
@@ -27,6 +27,12 @@ import type { QwikManifest, ResolvedManifest } from '@builder.io/qwik/optimizer'
 import { IsQData, QDATA_JSON, QDATA_JSON_LEN } from './user-response';
 import { isPromise } from './../../runtime/src/utils';
 
+// TODO: create single QGlobal type
+type AsyncStore = import('node:async_hooks').AsyncLocalStorage<RequestEventInternal>;
```

Comment:
Why are you doing a dynamic import here? Why not just regular static import? See https://developers.cloudflare.com/workers/runtime-apis/nodejs/asynclocalstorage/

Reply:
Because if it's running somewhere without ALS, it can behave as before